account_invoice_crm_tag/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Tecnativa - Stefan Ungureanu
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Invoice CRM Tag",
+    "version": "15.0.1.0.1",
+    "category": "Accounting",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "license": "AGPL-3",
+    "depends": ["account", "crm", "sale"],
```

Comment:
No need of such dependencies, as the tags in the sale.order belongs to `sale`, and `sale` already depends on `account`:

```suggestion
    "depends": ["sale"],
```